improver_tests/synthetic_data/test_set_up_test_cubes.py

```diff
@@ -428,54 +433,54 @@ def test_error_unmatched_realizations(self):
-                self.data_3d, height_levels=np.arange(height_levels_len)
+                self.data_3d, vertical_levels=np.arange(vertical_levels_len)
             )
 
-    def test_realizations_from_data_height_levels(self):
+    def test_realizations_from_data_verticals_levels(self):
         """Tests realizations from data and height coordinates added"""
-        height_levels = [1.5, 3.0, 4.5]
+        vertical_levels = [1.5, 3.0, 4.5]
         data_4d = np.array([self.data_3d, self.data_3d])
-        result = set_up_variable_cube(data_4d, height_levels=height_levels)
+        result = set_up_variable_cube(data_4d, vertical_levels=vertical_levels)
         self.assertArrayAlmostEqual(result.data, data_4d)
         self.assertEqual(result.coord_dims("realization"), (0,))
         self.assertArrayEqual(result.coord("realization").points, np.array([0, 1]))
         self.assertEqual(result.coord_dims("height"), (1,))
-        self.assertArrayEqual(result.coord("height").points, np.array(height_levels))
+        self.assertArrayEqual(result.coord("height").points, np.array(vertical_levels))
         self.assertEqual(result.coord_dims("latitude"), (2,))
         self.assertEqual(result.coord_dims("longitude"), (3,))
 
-    def test_realizations_height_levels(self):
+    def test_realizations_vertical_levels(self):
         """Tests realizations and height coordinates added"""
         realizations = [0, 3]
-        height_levels = [1.5, 3.0, 4.5]
+        vertical_levels = [1.5, 3.0, 4.5]
         data_4d = np.array([self.data_3d, self.data_3d])
         result = set_up_variable_cube(
-            data_4d, realizations=realizations, height_levels=height_levels
+            data_4d, realizations=realizations, vertical_levels=vertical_levels
         )
         self.assertArrayAlmostEqual(result.data, data_4d)
         self.assertEqual(result.coord_dims("realization"), (0,))
         self.assertArrayEqual(
             result.coord("realization").points, np.array(realizations)
         )
         self.assertEqual(result.coord_dims("height"), (1,))
-        self.assertArrayEqual(result.coord("height").points, np.array(height_levels))
+        self.assertArrayEqual(result.coord("height").points, np.array(vertical_levels))
         self.assertEqual(result.coord_dims("latitude"), (2,))
         self.assertEqual(result.coord_dims("longitude"), (3,))
 
-    def test_error_no_height_levels_4d_data(self):
-        """Tests error is raised if 4d data provided but not height_levels"""
+    def test_error_no_vertical_levels_4d_data(self):
+        """Tests error is raised if 4d data provided but not vertical_levels"""
         data_4d = np.array([self.data_3d, self.data_3d])
-        msg = "Height levels must be provided if data has > 3 dimensions."
+        msg = "Vertical levels must be provided if data has > 3 dimensions."
         with self.assertRaisesRegex(ValueError, msg):
             _ = set_up_variable_cube(data_4d)
```

Comment:
This may change with the new exceptions added for the if statement in the set up cubes function. I've not looked at the order, but it will become clear when you run the tests.